src/test/java/tests/auth/UserAuthServiceTest.java

```diff
@@ -122,4 +123,19 @@ public void userInvalidateToken() {
                 .body("httpStatus", equalTo("OK"))
                 .body("isSuccess", equalTo(true));
     }
+    @Test()
+    public void testUserInvalidRefreshTokenForAccessTokenCreation(){
+        Token token = Helper.getUserToken(userCredentials);
+        RefreshToken refreshToken = new RefreshToken();
+        refreshToken.setRefreshToken(token.getRefreshToken());
+        UserAuthEndpoints.userInvalidateToken(token.getAccessToken(), refreshToken);
+        Response response=InstitutionAuthEndpoints.adminTokenRefresh(refreshToken);
```

Comment:
Kod sanırım formatlanmamış;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html